.editorconfig

```diff
@@ -0,0 +1,96 @@
+﻿# CA1848: Use the LoggerMessage delegates
```

Comment:
do we have any precedence of all these disabled rule checks? has anyone approved of this? obviously I can't approve of this

Reply:
Done.